from0k2bp.tex

```diff
@@ -589,24 +599,24 @@
 a lot of other high quality resources too.
 
 As a preparation for using these ideas in practice, here is how the
-proof works for a (slightly) simpler case, that of Schnorr's identity
-protocol (which is basically the same as the Schnorr signature, except
-the interactive form, and ignoring messages). To review, the basic
+proof works for a (slightly) simpler case, namely for Schnorr's identity
+protocol\footnote{It is basically the same as the Schnorr signature, except
+the interactive form and ignoring messages}. To review, the basic
 structure is:
 
 Prover starts with a public key $P$ and a corresponding private key $x$ s.t. $P=xG$.
 
 Prover wishes to prove in zero knowledge, that he knows $x$.
 
-P → V: $R$ (a new random curve point, but P knows $k$ s.t. $R=kG$)
+$\mathcal{P}$ → $\mathcal{V}$: $R$ (a new random curve point, but $\mathcal{P}$ knows $k$ s.t. $R=kG$)
```

Comment:
I think `\mathcal` is a very good choice here, but we'll have to apply it to the introduction to section 3 also for consistency.

Reply:
Section 3 was updated.